helmfile/values/grafana-ops.yaml.gotmpl

```diff
@@ -20,3 +20,28 @@ dashboards:
     enabled: true
   networkpolicy:
     enabled: true
+  daily:
+    enabled: true
+
+daily:
```

Comment:
Shouldn't this be under `dashboards.daily`? That's at least how it is for `gatekeeper` and `falco`.